.github/workflows/build_jammy_humble_essentials.yml

```diff
@@ -0,0 +1,74 @@
+name: Docker - Build Jammy/Humble Essentials
+
+on:
+  pull_request:
+    branches: [ "main" ]
+    paths:
+      - ".github/workflows/**"
```

Comment:
Here, I would add a second line `"./devcontainer/**"` - meaning any PR that touches files within `devcontainer` will trigger this workflow.

Reply:
Done.